File: VTVApp.Api/Models/DTOs/Inspections/InspectionListDto.cs
namespace VTVApp.Api.Models.DTOs.Inspections
{
    public class InspectionListDto
    {
        public Guid Id { get; set; }
        public Guid VehicleId { get; set; }
        public string LicensePlate { get; set; }  // If VIN is needed for a quick reference
        public DateTime InspectionDate { get; set; }
        public int Status { get; set; } // Open, Completed, Failed, etc.
    }
}

File: VTVApp.Web/src/types/dtos/Inspections/InspectionListDto.ts
import { InspectionCheckpointDetailDto } from "./InspectionCheckpointDetailDto";

export interface InspectionListDto {
  id: string;
  vehicleId: string;
  licensePlate: string;
  inspectionDate: string;
  checkpoints: InspectionCheckpointDetailDto[]
  status: number;
}
